### PR TITLE
Bootstrap

### PR DESCRIPTION
**Pull request by mariadb-LeonidFedorov**
Revised bootstrap script

### CODE REVIEW

File: build/bootstrap_mcs.sh [hunk 1]
```diff
@@ -1,179 +1,215 @@
...
+
+source $SCRIPT_LOCATION/utils.sh
 
-if [ -z "$(grep mysql /etc/passwd)" ]; then
-    echo "Adding user mysql into /etc/passwd"
-    useradd -r -U mysql -d /var/lib/mysql
-    exit 1
+message "Building Mariadb Server from $MDB_SOURCE_PATH"
+
+BUILD_TYPE_OPTIONS=("Debug" "Release" "RelWithDebInfo" "MinSizeRel")
```
Comment (drrtuy) [inline build/bootstrap_mcs.sh]: We never build or test not MinSizeRel nor Release builds. I vaguely recall Release build fails to run in some way.

File: build/bootstrap_mcs.sh [hunk 2]
```diff
@@ -1,179 +1,215 @@
...
+                     -DPLUGIN_ROCKSDB=NO
+                     -DPLUGIN_TOKUDB=NO
+                     -DPLUGIN_CONNECT=NO
+                     -DPLUGIN_SPIDER=NO
+                     -DPLUGIN_OQGRAPH=NO
+                     -DPLUGIN_SPHINX=NO
+                     -DBUILD_CONFIG=mysql_release
+                     -DWITH_WSREP=OFF
+                     -DWITH_SSL=system
+                     -DWITH_UNITTESTS=YES
```
Comment (drrtuy) [inline build/bootstrap_mcs.sh:120]: WITH_MICROBENCHMARKS should be commented out by default b/c it calls for Google benchmark lib additional dependencies.

Comment (mariadb-LeonidFedorov) [reply build/bootstrap_mcs.sh:120]: And it should be OK! Unittest must be compiled by default, and this extra deps should be also

File: build/bootstrap_mcs.sh (hunk 1 above)
Comment (mariadb-LeonidFedorov) [reply build/bootstrap_mcs.sh]: Debug and RelWithDebInfo left

File: build/bootstrap_mcs.sh (hunk 2 above)
Comment (mariadb-LeonidFedorov) [reply build/bootstrap_mcs.sh:120]: WITH_MICROBENCHMARKS removed

**Review by drrtuy — APPROVED** [review]

**Pull request merged**